modules/codegen/src/main/scala/com/twilio/guardrail/generators/EndpointsGenerator.scala

```diff
@@ -0,0 +1,195 @@
+          case "401" => Target.pure((401, q"Unauthorized"))
+          case "402" => Target.pure((402, q"PaymentRequired"))
+          case "403" => Target.pure((403, q"Forbidden"))
+          case "404" => Target.pure((404, q"NotFound"))
+          case "405" => Target.pure((405, q"MethodNotAllowed"))
+          case "406" => Target.pure((406, q"NotAcceptable"))
+          case "407" => Target.pure((407, q"ProxyAuthenticationRequired"))
+          case "408" => Target.pure((408, q"RequestTimeout"))
+          case "409" => Target.pure((409, q"Conflict"))
+          case "410" => Target.pure((410, q"Gone"))
+          case "411" => Target.pure((411, q"LengthRequired"))
+          case "412" => Target.pure((412, q"PreconditionFailed"))
+          case "413" => Target.pure((413, q"RequestEntityTooLarge"))
+          case "414" => Target.pure((414, q"RequestUriTooLong"))
+          case "415" => Target.pure((415, q"UnsupportedMediaType"))
+          case "416" => Target.pure((416, q"RequestedRangeNotSatisfiable"))
+          case "417" => Target.pure((417, q"ExpectationFailed"))
+          case "418" => Target.pure((418, q"ImATeapot"))
+          case "420" => Target.pure((420, q"EnhanceYourCalm"))
+          case "422" => Target.pure((422, q"UnprocessableEntity"))
+          case "423" => Target.pure((423, q"Locked"))
+          case "424" => Target.pure((424, q"FailedDependency"))
+          case "425" => Target.pure((425, q"UnorderedCollection"))
+          case "426" => Target.pure((426, q"UpgradeRequired"))
+          case "428" => Target.pure((428, q"PreconditionRequired"))
+          case "429" => Target.pure((429, q"TooManyRequests"))
+          case "431" => Target.pure((431, q"RequestHeaderFieldsTooLarge"))
+          case "449" => Target.pure((449, q"RetryWith"))
+          case "450" => Target.pure((450, q"BlockedByParentalControls"))
+          case "451" => Target.pure((451, q"UnavailableForLegalReasons"))
+
+          case "500" => Target.pure((500, q"InternalServerError"))
+          case "501" => Target.pure((501, q"NotImplemented"))
+          case "502" => Target.pure((502, q"BadGateway"))
+          case "503" => Target.pure((503, q"ServiceUnavailable"))
+          case "504" => Target.pure((504, q"GatewayTimeout"))
+          case "505" => Target.pure((505, q"HTTPVersionNotSupported"))
+          case "506" => Target.pure((506, q"VariantAlsoNegotiates"))
+          case "507" => Target.pure((507, q"InsufficientStorage"))
+          case "508" => Target.pure((508, q"LoopDetected"))
+          case "509" => Target.pure((509, q"BandwidthLimitExceeded"))
+          case "510" => Target.pure((510, q"NotExtended"))
+          case "511" => Target.pure((511, q"NetworkAuthenticationRequired"))
+          case "598" => Target.pure((598, q"NetworkReadTimeout"))
+          case "599" => Target.pure((599, q"NetworkConnectTimeout"))
+          case _     => Target.raiseError(s"Unknown HTTP type: ${key}")
+        }
```

Comment:
Isn't this the third copy-paste of this code?  Can we move it somewhere common?

Reply:
Exceedingly unfortunately, not directly, as these are the underlying names of the status code objects in each library. akka-http uses `OK`, for example, where http4s uses `Ok`.

Reply:
So close, and yet so far...